book/website/collaboration/new-community/new-community-differences.md

```diff
@@ -0,0 +1,33 @@
+(cm-new-community-differences)=
+# Valuing Diversity and Differences
+
+When people collaborate with others, no matter if already have worked with them or they are complete strangers, you need to take into account that everyone will have different backgrounds, different cultures, different time zones, different levels of skills and very different lived experiences.
+
+## Welcome diverse perspectives from your members
+
+Think carefully about who your stakeholders are, if they are given fair opportunities to be heard, if certain groups are less represented than others, and if you have strategy in place to ensure diversity in your community.
+
+It is important to be mindful of the varying motivations of your members will have behind joining to work on your project.
+For example, in the case of COVID-19 related projects, you can expect that some people will join you because they want to know what kind of data on COVID-19 exist, some will come with an intention to be part of a group while learning some skills, some will simply hang out to watch what you are doing, whereas some people will join you because they have relatives who are suffering from COVID-19 infection and they want to speed up the diagnostic process.
+Be prepared to welcome their diverse perspectives into your project.
+
+## People have different personal circumstances
+
+As an individual we may think that everyone has the same working conditions as us, such as access to the tools and resources or their work setup, whereas the reality is far from that.
+Our colleagues even in the same city may have different living situations, caring responsibilities, medical conditions and resources available to them.  
```

Comment:
```suggestion
Our colleagues even in the same city may have different living situations, caring responsibilities, medical conditions and resources available to them.
```

I removed the extra space, as it rendered the next line as a separate paragraph.

Reply:
(Alternatively, if it does need to be a new paragraph, then an extra line after this one should be added.)